extensions/datasource/deployment/src/main/java/io/quarkus/datasource/deployment/devservices/DevServicesDatasourceProcessor.java

```diff
@@ -262,8 +262,19 @@ public void run() {
                             dataSourceBuildTimeConfig.devservices.port, launchMode);
             closeableList.add(datasource.getCloseTask());
 
-            Map<String, String> devDebProperties = new HashMap<>();
             propertiesMap.put(prefix + "db-kind", dataSourceBuildTimeConfig.dbKind.orElse(null));
+            String devServicesPrefix = prefix + "devservices.";
+            if (dataSourceBuildTimeConfig.devservices.imageName.isPresent()) {
+                propertiesMap.put(devServicesPrefix + "image-name", dataSourceBuildTimeConfig.devservices.imageName.get());
+            }
+            if (dataSourceBuildTimeConfig.devservices.port.isPresent()) {
+                propertiesMap.put(devServicesPrefix + "port", dataSourceBuildTimeConfig.devservices.port.getAsInt());
+            }
+            if (!dataSourceBuildTimeConfig.devservices.properties.isEmpty()) {
+                propertiesMap.put(devServicesPrefix + "port", dataSourceBuildTimeConfig.devservices.properties);
```

Comment:
This line should probably read:
```java
propertiesMap.put(devServicesPrefix + "properties", dataSourceBuildTimeConfig.devservices.properties);
```

Reply:
Good catch @knutwannheden can you open a followup PR? Thanks

Reply:
Sure.

Reply:
Awesome, thanks a lot.